### ISSUE TITLE
Problem with 'PX' in Elements

### ISSUE THREAD

**Issue by Igcorreia** [Prinzhorn/skrollr#703]
Let's say I want to create a custom tag like:
The I want to animate 2 properties:
The problem is that Skroller is forcing values in pixels and I want the as integer.
<--- I just don't want the 'px' there.

**Comment by Prinzhorn** [comment]
> The problem is that Skroller is forcing values in pixels and I want the as integer.

it does not. If it would, then things like font-size and z-index couldn't be animated

**Comment by Igcorreia** [comment]
Yes something like that. Does that  work? Does it mean I can animate any property without using pixels? If that is the case, then there is no bug :)

-- Suggestion to make this easier to find in the docs.
